### DISCUSSION TITLE
Images not display

### DISCUSSION THREAD

**Discussion by HuaxiaGoodBoy** [flet-dev/flet#3888] [Q&A] [answered]
### Question

Images not displayed when using local path and setting AppView to "WEB_BROWSER"

### Code sample

```python
import flet as ft


def main(page: ft.Page):
    page.title = "GridView Example"
    page.theme_mode = ft.ThemeMode.DARK
    page.padding = 50
    page.update()

    images = ft.GridView(
        expand=1,
        runs_count=5,
        max_extent=200,
        child_aspect_ratio=1.0,
        spacing=50,
        run_spacing=50,

    )

    page.add(images)

    for img in ['img1.jpg', 'img2.jpg', 'img3.jpg', 'img4.jpg']:
        images.controls.append(
            ft.Image(
                src=f"./img/{img}",
                fit=ft.ImageFit.CONTAIN,
                repeat=ft.ImageRepeat.NO_REPEAT,
                border_radius=ft.border_radius.all(10),
                tooltip='Compressor',
            )
        )

    page.update()


ft.app(target=main, view=ft.WEB_BROWSER)
```

### Error message

```bash
Blank browser page
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Remove the first dot from the path and retry.

**Reply by HuaxiaGoodBoy** [reply]
It doesn't work either.



    for img in ['img1.jpg', 'img2.jpg', 'img3.jpg', 'img4.jpg']:
        images.controls.append(
            ft.Image(
                src=f"/img/{img}",
                fit=ft.ImageFit.CONTAIN,
                repeat=ft.ImageRepeat.NO_REPEAT,
                border_radius=ft.border_radius.all(10),
                tooltip='Compressor',
            )
        )

**Reply by ndonkoHenri** [reply]
where is the img folder located? (in the root folder of the app or in some other one)

**Comment by HuaxiaGoodBoy** [comment]
In the root folder.
<img width="698" alt="image" src="https://github.com/user-attachments/assets/483c1d9a-2ae0-46e2-966d-8c3aca133ac3">

**Reply by HuaxiaGoodBoy** [reply]
The imagses could display if removing "view=ft.FLET_APP_WEB" in line 36.

ft.app(target=main)

**Reply by ndonkoHenri** [reply, marked as answer]
Read this: https://flet.dev/docs/controls/image#src
You need to set the img folder as asset directory.

**Comment by HuaxiaGoodBoy** [comment]
It works well now. Thanks a lot for your guideline! 👍
